kano/utils/hardware.py

```diff
@@ -151,13 +226,8 @@ def has_min_performance(score):
         rv (bool) - True if the hardware has a higher score than the given or if the
                     hardware could not be detected; and False otherwise
     """
-    rv = True
-    model = get_rpi_model()
-
-    if model in PERFORMANCE_SCORES:
-        model_score = PERFORMANCE_SCORES[model]
 
-        if model_score < score:
-            rv = False
+    model = get_rpi_model()
+    model_score = get_board_property(model, 'performance')
 
-    return rv
+    return model_score and model_score >= score
```

Comment:
Make sure you preserve the previous behaviour here please. If the above does not have a `model_score` it returns `False` instead of `True`. The assumption made there was that new boards are going up in performance instead of down.